samples/api/hpp_hello_triangle_1_3/hpp_hello_triangle_1_3.cpp

```diff
@@ -0,0 +1,1162 @@
+#endif
+
+	if (!validate_extensions(required_instance_extensions, available_instance_extensions))
+	{
+		throw std::runtime_error("Required instance extensions are missing.");
+	}
+
+	std::vector<const char *> requested_instance_layers{};
+
+#if defined(VKB_DEBUG) || defined(VKB_VALIDATION_LAYERS)
+	char const *validationLayer = "VK_LAYER_KHRONOS_validation";
+
+	std::vector<vk::LayerProperties> supported_instance_layers = vk::enumerateInstanceLayerProperties();
+
+	if (std::ranges::any_of(supported_instance_layers, [&validationLayer](auto const &lp) { return strcmp(lp.layerName, validationLayer) == 0; }))
+	{
+		requested_instance_layers.push_back(validationLayer);
+		LOGI("Enabled Validation Layer {}", validationLayer);
+	}
+	else
+	{
+		LOGW("Validation Layer {} is not available", validationLayer);
+	}
+#endif
+
+	vk::ApplicationInfo app{.pApplicationName = "Hello Triangle V1.3", .pEngineName = "Vulkan Samples", .apiVersion = VK_MAKE_VERSION(1, 3, 0)};
+
+	vk::InstanceCreateInfo instance_info{.pApplicationInfo        = &app,
+	                                     .enabledLayerCount       = static_cast<uint32_t>(requested_instance_layers.size()),
+	                                     .ppEnabledLayerNames     = requested_instance_layers.data(),
+	                                     .enabledExtensionCount   = static_cast<uint32_t>(required_instance_extensions.size()),
+	                                     .ppEnabledExtensionNames = required_instance_extensions.data()};
+
+#if defined(VKB_DEBUG) || defined(VKB_VALIDATION_LAYERS)
+	vk::DebugUtilsMessengerCreateInfoEXT debug_messenger_create_info;
+	if (has_debug_utils)
+	{
+		debug_messenger_create_info.messageSeverity = vk::DebugUtilsMessageSeverityFlagBitsEXT::eError | vk::DebugUtilsMessageSeverityFlagBitsEXT::eWarning;
+		debug_messenger_create_info.messageType     = vk::DebugUtilsMessageTypeFlagBitsEXT::eGeneral | vk::DebugUtilsMessageTypeFlagBitsEXT::eValidation;
+		debug_messenger_create_info.pfnUserCallback = debug_callback;
+
+		instance_info.pNext = &debug_messenger_create_info;
+	}
+#endif
+
+#if defined(VKB_ENABLE_PORTABILITY)
+	if (portability_enumeration_available)
+	{
+		instance_info.flags |= vk::InstanceCreateFlagBits::eEnumeratePortabilityKHR;
+	}
+#endif
+
+	// Create the Vulkan instance
+	context.instance = vk::createInstance(instance_info);
+
+	// initialize function pointers for instance
+	VULKAN_HPP_DEFAULT_DISPATCHER.init(context.instance);
+
+#if defined(VKB_DEBUG) || defined(VKB_VALIDATION_LAYERS)
+	if (has_debug_utils)
+	{
+		context.debug_callback = context.instance.createDebugUtilsMessengerEXT(debug_messenger_create_info);
+	}
+#endif
+
+#if defined(VK_USE_PLATFORM_DISPLAY_KHR) || defined(VK_USE_PLATFORM_ANDROID_KHR) || defined(VK_USE_PLATFORM_METAL_EXT)
+	// we need some additional initializing for this platform!
+	if (volkInitialize())
+	{
+		throw std::runtime_error("Failed to initialize volk.");
+	}
+	volkLoadInstance(instance);
```

Comment:
context.instance, not instance. With that changed, this seems to work now.

Reply:
Bummer!
I should have compiled that, at least.